Authors:
 Luca Bruno <lucab@debian.org>
 Alexander Kurtz <kurtz.alex@googlemail.com>
 Daniel Baumann <daniel.baumann@progress-technologies.net>
Description:
 Patch to properly cast and avoiding compiler warnings. Fixes FTBFS on alpha
 and ia64 (Closes: #539433, #583768).

--- a/btrfs-convert.c
+++ b/btrfs-convert.c
@@ -2444,7 +2444,7 @@ static int do_rollback(const char *devname, int force)
 	ext2_root = btrfs_read_fs_root(root->fs_info, &key);
 	if (!ext2_root || IS_ERR(ext2_root)) {
 		fprintf(stderr, "unable to open subvol %llu\n",
-			key.objectid);
+			(unsigned long long) key.objectid);
 		goto fail;
 	}
 
